src/DotNet.Consolidate.Tests/DotNet.Consolidate.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
The same here, the tool is pretty simple and doesn't have any specific requirements for frameworks. So we don't need to run the test against multiple frameworks. The latest one should be enough.

Reply:
Changed it to only target .net 8